### ISSUE TITLE
Update whitepaper

### ISSUE THREAD

**Issue by RafaelAPB** [hyperledger/cacti#2691]
Our whitepaper is around two years old and no longer reflects the project's current state. I'd like to suggest some coordination amongst maintainers and contributors of Cacti to update the whitepaper.  While not a core distinguished of our project, I believe we should make this a priority, not only to provide better visibility of our project but also to document the rationale behind our project.  

cc @petermetz

**Comment by petermetz** [comment]
@RafaelAPB I agree! With the v2.0.0 release coming up and the workshop in November being so close as well, I can't volunteer any of my time for the whitepaper renovation unfortunately.
If anyone else has the time to do it, great, but if no one does, then I'd suggest that we just remove it for now so that it does not confuse people (confusing documentation is worse than not having as much documentation to begin with)

**Comment by RafaelAPB** [comment]
Agreed @petermetz - let's do it after this next release.

**Comment by petermetz** [comment]
> Agreed @petermetz - let's do it after this next release.

@RafaelAPB FYI: I sent this in the meantime: https://github.com/hyperledger/cacti/pull/2699

**Comment by VRamakrishna** [comment]
@petermetz @RafaelAPB I can help in this task, though like you both, it's going to be tough to extract too many cycles for it from my schedule.

I am currently refactoring the integrated documentation (legacy Cactus + legacy Weaver) and publishing a MkDocs version, the PR for which I will submit in the next few days. Based on what the integrated docs contain, I can (at some point in October), propose a new whitepaper outline (i.e., table of contents). We can then discuss it and then divide the tasks among ourselves.

I think we ought to schedule dedicated meetings on the Cacti calendar to brainstorm the details, though (fair warning) I'll be out of action for a good chunk of November and December.

**Comment by VRamakrishna** [comment]
For your reference, here is the documentation PR I just opened: https://github.com/hyperledger/cacti/pull/2726.

**Comment by RafaelAPB** [comment]
Let us discuss this through the proposed schedule. Thank you for your input @VRamakrishna.

**Comment by petermetz** [comment]
> @petermetz @RafaelAPB I can help in this task, though like you both, it's going to be tough to extract too many cycles for it from my schedule.
> 
> I am currently refactoring the integrated documentation (legacy Cactus + legacy Weaver) and publishing a MkDocs version, the PR for which I will submit in the next few days. Based on what the integrated docs contain, I can (at some point in October), propose a new whitepaper outline (i.e., table of contents). We can then discuss it and then divide the tasks among ourselves.
> 
> I think we ought to schedule dedicated meetings on the Cacti calendar to brainstorm the details, though (fair warning) I'll be out of action for a good chunk of November and December.

Thank you @VRamakrishna I agree that we could add the whitepaper to the maintainer meeting agenda, but until the workshop is finished I definitely won't be able to work on it unfortunately (maybe some parts that are specific to the workshop that I'll be able to use there but probably not even that because first and foremost I want the code to work)

**Comment by RafaelAPB** [comment]
Kindly let me know when we schedule a maintainers meeting to address this.

**Comment by petermetz** [comment]
> Kindly let me know when we schedule a maintainers meeting to address this.

@RafaelAPB For me, the best would be in December!
